examples/ads.amp.html

```diff
@@ -582,6 +583,13 @@ <h2 id="custom">Custom leaderboard</h2>
       data-eas-cu="3255">
   </amp-ad>
 
+  <h2>Dable</h2>
+  <amp-embed width="450" height="140"
+      type="dable"
+      data-widget_id="jobgqR7W"
```

Comment:
format `data-widget-id` is preferred. you can read it from data object using the camel format:

`data.widgetId`